WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -148,20 +159,154 @@ - (void)viewDidLoad
+}
+
+- (BlogDetailsSection *)publishTypeSectionViewModel
+{
+    __weak __typeof(self) weakSelf = self;
+    NSMutableArray *rows = [NSMutableArray array];
+    [rows addObject:[[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Blog Posts", @"Noun. Title. Links to the blog's Posts screen.")
+                                                    image:[UIImage imageNamed:@"icon-menu-posts"]
+                                                 callback:^{
+                                                     [weakSelf showPostListForBlog];
+                                                 }]];
+
+    [rows addObject:[[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Pages", @"Noun. Title. Links to the blog's Pages screen.")
+                                                    image:[UIImage imageNamed:@"icon-menu-pages"]
+                                                 callback:^{
+                                                     [weakSelf showPageListForBlog];
+                                                 }]];
+
+    BlogDetailsRow *row = [[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Comments", @"Noun. Title. Links to the blog's Comments screen.")
+                                                          image:[UIImage imageNamed:@"icon-menu-comments"]
+                                                       callback:^{
+                                                           [weakSelf showCommentsForBlog];
+                                                       }];
+    NSUInteger numberOfPendingComments = [self.blog numberOfPendingComments];
+    if (numberOfPendingComments > 0) {
+        row.detail = [NSString stringWithFormat:@"%d", numberOfPendingComments];
+    }
+    [rows addObject:row];
+
+    NSString *title = NSLocalizedString(@"Publish", @"Section title for the publish table section in the blog details screen");
+    return [[BlogDetailsSection alloc] initWithTitle:title andRows:rows];
+}
+
+- (BlogDetailsSection *)appearanceSectionViewModel
+{
+    __weak __typeof(self) weakSelf = self;
+    NSMutableArray *rows = [NSMutableArray array];
+    [rows addObject:[[BlogDetailsRow alloc] initWithTitle:NSLocalizedString(@"Themes", @"Themes option in the blog details")
+                                                    image:[UIImage imageNamed:@"icon-menu-theme"]
+                                                 callback:^{
+                                                     [weakSelf showThemesForBlog];
+                                                 }]];
+
+    NSString *title =NSLocalizedString(@"Personalize", @"Section title for the personalize table section in the blog details screen.");
+    return [[BlogDetailsSection alloc] initWithTitle:title andRows:rows];
+}
+
+- (BlogDetailsSection *)configurationSectionViewModel
+{
+    __weak __typeof(self) weakSelf = self;
+    NSMutableArray *rows = [NSMutableArray array];
+
+#ifdef WP_PEOPLE_ENABLED
```

Comment:
Can you use the feature flag here? I didn't originally because it relied on a constant so I needed to check at compile time